resourcehandler/resourcehandler.go

```diff
@@ -25,62 +25,62 @@ type ResourceChange struct {
 	Changes string
 }
 
-// Sync synchornize resources with Spinnaker
-type Sync struct {
+// ResourceHandler synchornize resources with Spinnaker
```

Comment:
I don't like the name. `Sync` indicated that this structure was responsible for resource synchronization. And indeed, its main responsibility was to synchronize resources. I assume that by “ResourceHandler” you meant a structure which handles the resources. Perhaps `ResourceManager` or `ResourceDirector` would be a better name.

Reply:
Good point, I'll go with `ResourceManager`.